resources/migrations/20180426153251-update-issue-execution-posted.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."issues" ADD COLUMN "execute_posted" boolean;
```

Comment:
Where is this field used?

Reply:
Nice catch, thank you! My original implementation for revocation was perhaps overly explicit in the steps underneath the hood that were being performed. It has since been agreed upon that the UI should contain as few steps as possible, so I should rip out these legacy columns.